Rakefile

```diff
@@ -98,3 +102,31 @@ task :release do
     end
   end
 end
+
+namespace :local do
+  desc "show help"
+  task :version do
+    require_relative 'mrblib/mruby-cli/version'
+    puts "mruby-cli #{MRubyCLI::Version::VERSION}"
+  end
+end
+
+def is_in_a_docker_container?
+  `grep -q docker /proc/self/cgroup`
+  $?.success?
+end
+
+Rake.application.tasks.each do |task|
+  next if ENV["RUN_RAKE_TASK_LOCALLY"]
```

Comment:
I like the idea to have something prefixing with `MRUBY_CLI`. What about `MRUBY_CLI_LOCAL_BUILD`?

As the last suggestion is very generic, it would be useful for other values. Would it mean we have to deal with comma-separated values? IMK, it's quite uncommon.